_build/data/transport.core.system_settings.php

```diff
@@ -501,6 +501,7 @@
   'area' => 'authentication',
   'editedon' => null,
 ), '', true, true);
+/*
```

Comment:
Instead of commenting out things that are no longer needed, just remove them completely. We have the git history to go back if needed.